docs/src/conf.py

```diff
@@ -404,6 +404,7 @@ def _dotv(version):
     "https://twitter.com/scitools_iris",
     "https://stackoverflow.com/questions/tagged/python-iris",
     "https://www.flaticon.com/",
+    "https://www.freepik.com/",
```

Comment:
This is coming up as "forbidden", which suggests that linkcheck doesn't want to touch it, rather than it being broken, so that feels permanent hence me adding it to the list.